tests/test_runners/test_azure_runner.py

```diff
@@ -183,6 +183,34 @@ def test_show_terraform_outputs(
             assert output in captured.out
 
 
+def test_provision(matcha_testing_directory: str, template_runner: AzureRunner):
+    """Test service can provision resources using terraform.
+
+    Args:
+        matcha_testing_directory (str): Testing directory
+        template_runner (AzureRunner): a AzureRunner object instance
+    """
+    os.chdir(matcha_testing_directory)
+    template_runner._check_terraform_installation = MagicMock()
+    template_runner._validate_terraform_config = MagicMock()
+    template_runner._initialize_terraform = MagicMock()
+    template_runner._apply_terraform = MagicMock()
+    template_runner._show_terraform_outputs = MagicMock()
+
+    # Create mock matcha.state file with the required contents
```

Comment:
I think this comment may be incorrect?

Reply:
Removed, along with the one in`test_deprovision`.